marklogic-client-api/src/test/java/com/marklogic/client/test/dataservices/CallBatcherDefaultArgsTest.java

```diff
@@ -321,10 +321,6 @@ public void singleAtomicDifferentParamsTest() {
                     fail("atomic batcher failed");
                 });
 
```

Comment:
If the test doesn't provide any data, what is being tested?  Do I misunderstand?

Reply:
The "twoDiffAtomic" endpoint in singleAtomicDifferentParamsTest is configured for taking two parameters, first a double (multiple) which is not nullable and a float value(not multiple) which can be nullable. So this test case is asserting that no exception is thrown when a null float value is passed as second parameter. 
The next test case singleAtomicDifferentParamsTest2 throws an exception on passing a null float value because there the second parameter is configured to be not nullable.